### ISSUE TITLE
Fatal error: Cannot redeclare parseArgs(), psysh 0.10.2 not working

### ISSUE THREAD

**Issue by proggga** [bobthecow/psysh#620]
Fatal error: Cannot redeclare parseArgs() (previously declared in phar:///home/mfesenko/psysh/vendor/autoload.php:20) in /home/mfesenko/data/www/lib/cli.lib.php on line 45

please tell me what I'm doing wrong, or this is bug
I have include in 'defaultIncludes'
and it has same function parseArgs, but it fail, so I can't use 10.2

Fatal error: Cannot redeclare parseArgs() (previously declared in phar:///home/mfesenko/psysh/vendor/autoload.php:20) in /home/mfesenko/data/www/lib/cli.lib.php on line 45

Call Stack:
    0.0003     407768   1. {main}() /home/mfesenko/psysh:0
    0.0233    1730952   2. Psy\{closure:phar:///home/mfesenko/psysh/src/functions.php:125-207}() /home/mfesenko/psysh:145
    0.0444    5020840   3. Psy\Shell->run() phar:///home/mfesenko/psysh/src/functions.php:203
    0.0518    5277904   4. Psy\Shell->run() phar:///home/mfesenko/psysh/src/Shell.php:149
    0.0562    5280304   5. Psy\Shell->doRun() phar:///home/mfesenko/psysh/vendor/symfony/console/Application.php:96
    0.0562    5280304   6. Psy\Shell->doInteractiveRun() phar:///home/mfesenko/psysh/src/Shell.php:162
    0.0906    7392872   7. Psy\Shell->loadIncludes() phar:///home/mfesenko/psysh/src/Shell.php:176
    0.0906    7393192   8. Psy\Shell->Psy\{closure:phar:///home/mfesenko/psysh/src/Shell.php:204-219}() phar:///home/mfesenko/psysh/src/Shell.php:220
    0.3038   43138488   9. include('myinlcude.lib.php') phar:///home/mfesenko/psysh/src/Shell.php:208
    0.9460  148180840  10. require_once('../somelib.lib.php') ...somelib.lib.php:26

**Comment by bobthecow** [comment]
Did this work in previous versions of psysh?

Can you share your psysh config?

**Comment by proggga** [comment]
this version works well
Psy Shell v0.9.12 (PHP 7.2.22-1+0~20190902.26+debian8~1.gbpd64eb7 - cli) by Justin Hileman


I only removed paths, because cause of confidentiality agreement

this my config
```php
<?php
return [
    'defaultIncludes' => [
        'somelib.php',
        'somelib2.lib.php',
    ],
    'errorLoggingLevel' => E_ALL & ~E_NOTICE,
    'useUnicode' => false,
];
```
And yes, I have function 'parseArgs' in my project

**Comment by proggga** [comment]
Also I remove includes and test this one, how you can see, It's not allowed to use this function in new version of psysh 
![image](https://user-images.githubusercontent.com/12262156/78153919-45733e00-7444-11ea-81fa-0379b32dc350.png)

**Comment by bobthecow** [comment]
Gotcha. Yep, that does seem to be a problem. Thanks for reporting :)

**Comment by proggga** [comment]
Found that all functions from vendor/bin/php-parse or maybe from vendor/nikic/php-parser/bin/php-parse somehow already in global namespace
function showHelp() too

**Comment by bobthecow** [comment]
yeah, i’ve got a fix.

**Comment by bobthecow** [comment]
This is fixed in 032449c, and will be in the prebuilt pharos in the next stable build. In the meantime, you can build your own phar, or you can use one of the composer install methods to work around it.

Thanks again for reporting!
